### PR TITLE
Added support to Screen Space UI camera type.

### PR DESCRIPTION
**Pull request by phi-lira**
### Purpose of this PR
 Camera Stacking solution applies post-processing at the end of the stack. This caused an undesirable behavior of users wanting to composite UI with an overlay camera.
 This PR exposes another camera render type `Screen Space UI` to render UI after Post-processing. 
 This is a very lean rendering camera that focuses on rendering UI only. (no occlusion culling, no lighting culling, no depth or color copy or shadow map)

---
### Testing status

**Manual Tests**: What did you do?
- [x ] Opened test project + Run graphic tests locally
- [x ] Built a player
- [x ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [x ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

---
### Comments to reviewers

UICamera

### CODE REVIEW

File: com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs [hunk 1]
```diff
@@ -281,17 +281,27 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
...
+            var renderer = baseCameraAdditionalData?.scriptableRenderer;
+            bool supportsCameraStacking = renderType == CameraRenderType.Base && renderer != null && renderer.supportedRenderingFeatures.cameraStacking;
+            List<Camera> cameraStack = (supportsCameraStacking) ? baseCameraAdditionalData?.cameraStack : null;
 
             // We need to know the last active camera in the stack to be able to resolve
             // rendering to screen when rendering it. The last camera in the stack is not
             // necessarily the last active one as it users might disable it.
             int lastActiveOverlayCameraIndex = -1;
-            if (cameraStack != null)
+            if (cameraStack != null && baseCameraAdditionalData.renderType == CameraRenderType.Base)
```
Comment (ellioman) [inline com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs]: Do we need this base camera test? As cameraStack is null if it's not base...

**Comment by ellioman** [comment]
Closing this one as we have another solution.
